WellCoffeeInventoryAPI/src/main/java/com/team_lightyear/WellCoffeeInventoryAPI/models/Item.java

```diff
@@ -62,20 +63,29 @@ public class Item {
     public Item() {
     }
 
+        //Constructor String
```

Comment:
Could you explain this constructor?

Reply:
I'm not certain how that got there. I think I had kept that in case I needed to change the field type of comment. Delete now.